eth/main.cpp

```diff
@@ -405,403 +315,556 @@ int main(int argc, char** argv)
 	bool chainConfigIsSet = false;
 	string configJSON;
 	string genesisJSON;
-	for (int i = 1; i < argc; ++i)
+	po::options_description clientDefaultMode("Client mode (default)");
+	clientDefaultMode.add_options()
+			("format", po::value<string>(), "<format> Set format.")
```

Comment:
I would prefer these lines to be one tab indented, not 2 tabs like you did ( I don't see the reason to make it unusual 2 tabs and it can lead to some additional line breaks, which makes it harder to read)